src/components/TrombinoAddPeople.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { Link } from "react-router";
+import Form from "react-jsonschema-form";
+
+
+export default class TrombinoAddPage extends Component {
+  addPeople(form) {
+    console.log(form.formData);
+    //this.props.trombinoAddPeople(this.props.server, this.props.bucket, form.formData);
+  }
+  
+  render() {
+    const {
+      title,
+      companies,
+      form
+    } = this.props;
+
+    if (!form) return <div>Loading</div>;
+
+    form.uiSchema['company'] = {
+      "ui:widget": (props) => {
+        console.log(props);
+        return (
```

Comment:
@n1k0 I don't know what I am missing there, but the value of the field don't change when I change the selected option. I tried doing onChange={props.onChange} but I'd got an error.

Any clue?